.gitignore

```
@@ -1 +1,4 @@
 node_modules
+
+src/components/Atoms/.DS_Store
+src/components/.DS_Store
```

これだとディレクトリ追加するたびに書く必要が出てきて大変なので、正規表現を使ってこう書こう

```suggestion
**/.DS_Store
```